File: java/id202.java
import java.util.*;

public class id202 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();
        Set<Character> charSet = new HashSet<>();
        boolean flag = true;
        for (char c : str.toCharArray()) {
            if (!charSet.add(c)) {
                flag = false;
                break;
            }
        }
        if (flag) {
            System.out.println("YES");
        } else {
            System.out.println("NO");
        }
        sc.close();
    }
}

File: c++/easy_LV1/id202.cpp
#include <bits/stdc++.h>
using namespace std;
const int K = 1e3 + 5;
int a[K];
int main() {
    string str;
    cin >> str;
    int flag = 1;
    for (char s : str) {
        if (a[s - 0] != 0) {
            flag = 0;
            break;
        }
        a[s - 0] ++;
    }
    if (flag == 1) {
        cout << "YES" << "\n";
    } else {
        cout << "NO" << "\n";
    }
    return 0;
}